README.md

```diff
@@ -13,7 +13,8 @@ Python library with command line interface for building docker images.
  * build inside a docker container (so your builds are separated between each other)
  * git as a source to your Dockerfile (you may specify commit/branch and path to Dockerfile within the git repo)
  * collect build logs
- * integration with [koji](http://koji.fedoraproject.org/koji/) build system
+ * integration with
+   [koji](https://github.com/projectatomic/atomic-reactor/blob/master/docs/koji.md) build system
```

Comment:
It'd be nice to have this link being relative, so it behaves properly on forked projects: `../blob/master/docs/koji.md`
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

Although, it'd be even nicer if it was relative to the current branch, but not sure if that's supported.